rest/route/host_test.go

```diff
@@ -32,15 +32,23 @@ import (
 
 type HostsChangeStatusesSuite struct {
 	route *hostsChangeStatusesHandler
+	ctx   context.Context
```

Comment:
Does this need a context? It doesn't seem like it's ever used.

Reply:
Ah yeah, I forgot to change these tests to get rid of the excessive background context depedence in these tests. I changed them all to use cancelled contexts.